src/utils/service/service.js

```diff
@@ -159,36 +160,75 @@ class Service {
     this.globalCallbacks = false;
   }
 
+  axiosParams(options) {
```

Comment:
missing comment on this function for consistency with other functions in the class